spec/requests/graphql/car_monthly_mileage_request_spec.rb

```diff
@@ -36,6 +36,14 @@
           month
           year
           totalMileage
+          car {
+            id
+            make
+            model
+            year
+            mpg
+            fuelType
+          }
```

Comment:
Love this! One idea I had was giving a user the option of naming their car? Not incredibly important at all lol but could help with naming on the graph if we end up wanting to show it separated by car/input type and could be fun for the user, idk 🤷‍♀️

Reply:
Fo sho, I'm a fan of that idea!! That should be pretty easy to do, and it would be useful for that one person who for some reason has twelve of the same car or something XD